ajax/libs/leaflet-contextmenu/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "leaflet-contextmenu",
+  "filename": "leaflet.contextmenu.min.js",
```

Comment:
The **leaflet.contextmenu.js** has been minified from **leaflet.contextmenu-src.js**
So I think you can just minify **leaflet.contextmenu.css**
And change the ```"filename"``` to ```"leaflet.contextmenu.js"```

@PeterDaveHello please help me review my opinion.
I'm not sure whether I'm right.